Assets/Scripts/EnergyCounter.cs

```diff
@@ -32,9 +36,9 @@ void Start()
         UpdateEnergy();
     }
 
-    public void AddEnergy(int amount)
+    public void AddEnergy()
```

Comment:
We should keep the argument for this function in case Design wants to add other ways in which we gain energy.